brevo/README.md

```diff
@@ -0,0 +1,62 @@
+## Overview
+
+[Brevo][1] is a cloud-based digital marketing platform designed for creating, sending, and tracking email campaigns, transactional emails, and more. It offers tools for automation and analytics, helping businesses optimize their email marketing strategies and monitor performance.
+
+Integrate Brevo with Datadog to gain insights into Brevo marketing campaign emails and track Brevo performance based on events and other transactional events using [webhooks][2].
+
+## Setup
+
+Follow the instructions below to configure this integration for Brevo Marketing and Transactional events through a Webhook.
+
+### Configuration
+
+#### Webhook configuration for marketing events
+Configure the Datadog endpoint to forward Brevo marketing events as logs to Datadog. For more details, see the Brevo [Marketing webhooks][3] documentation.
```

Comment:
Here are the instructions I'm putting on the tile, can you please update them to match: 

```
### Webhook Configuration for Brevo Marketing Events

Configure the Datadog endpoint to forward Brevo marketing events as logs to Datadog. For more information, see Brevo's [How To Use Webhooks](https://developers.brevo.com/docs/how-to-use-webhooks) documentation.

1. Either select an existing API key or create a new API key by clicking one of the buttons below. Clicking either button will generate a URL for further action.
2. Log in to your [Brevo account](https://login.brevo.com/).
3. In the left navigation panel, go to **Campaigns**.
4. Open the **Settings** page.
5. In the **Webhooks** section, click **Configure**.
6. Click **Add a New Webhook**.
7. Enter the previously identified webhook URL.
8. Select the types of messages and contact logs to forward to Datadog.
9. Click **Add** to save.

### Webhook Configuration for Brevo Transactional Events

Set up the Datadog endpoint to forward Brevo transactional events as logs to Datadog. For more details, refer to the Brevo [Transactional Webhooks](https://developers.brevo.com/docs/transactional-webhooks) documentation.

1. Select an existing API key or create a new one by clicking one of the buttons below.
2. Log in to your [Brevo account](https://login.brevo.com/). (If already logged in, you'll be redirected to the [Brevo homepage](https://app.brevo.com/).)
3. In the left navigation panel, go to **Transactional**.
4. Open the **Settings** and select **Webhook**.
5. Click **Add a New Webhook**.
6. Enter the previously identified webhook URL.
7. Choose the types of message logs to forward to Datadog.
8. Click **Save** to finalize the setup.

```

Reply:
@apiazza-dd I think the links which is mentioned in configuration steps of Marketing events should point to marketing webhook settings page instead of how to use webhook page. 
Below is the revised content of what you shared. Please review and let us know in case of any changes.
Also, the links which are mentioned as [3], [4] is defined at the end of the README file. Hence it points to correct destination.

```
### Webhook Configuration for Brevo Marketing Events

Configure the Datadog endpoint to forward Brevo marketing events as logs to Datadog. For more information, see Brevo's [Marketing Webhooks][3] documentation.

1. Either select an existing API key or create a new API key by clicking one of the buttons below. Clicking either button will generate a URL for further action.
2. Log in to your [Brevo account][4].
3. In the left navigation panel, click **Campaigns**.
4. Open the **Settings** page.
5. In the **Webhooks** section, click **Configure**.
6. Click **Add a New Webhook**.
7. Enter the previously identified webhook URL.
8. Select the types of messages and contact logs to forward to Datadog.
9. Click **Add** to save.

### Webhook Configuration for Brevo Transactional Events

Set up the Datadog endpoint to forward Brevo transactional events as logs to Datadog. For more details, refer to the Brevo [Transactional Webhooks][5] documentation.

1. Select an existing API key or create a new one by clicking one of the buttons below.
2. Log in to your [Brevo account][4]. (If already logged in, you'll be redirected to the [Brevo homepage][6].)
3. In the left navigation panel, click **Transactional**.
4. Open the **Settings** and select **Webhook**.
5. Click **Add a New Webhook**.
6. Enter the previously identified webhook URL.
7. Choose the types of message logs to forward to Datadog.
8. Click **Save** to finalize the setup.


## Data Collected

### Logs
The Brevo integration forwards the marketing and transactional event logs to Datadog.

### Metrics
Brevo does not include any metrics.

### Service Checks
Brevo does not include any service checks.

### Events
Brevo does not include any events.

## Troubleshooting

Need help? Contact [Datadog support][7].

[1]: https://www.brevo.com/products/marketing-platform/
[2]: https://developers.brevo.com/docs/how-to-use-webhooks
[3]: https://developers.brevo.com/docs/marketing-webhooks
[4]: https://login.brevo.com/
[5]: https://developers.brevo.com/docs/transactional-webhooks
[6]: https://app.brevo.com/
[7]: https://docs.datadoghq.com/help/
```